hs_collection_resource/page_processors.py

```diff
@@ -0,0 +1,176 @@
+from django.db.models import Q
+from mezzanine.pages.page_processors import processor_for
+from crispy_forms.layout import Layout, HTML
+
+from hs_core import page_processors
+from hs_core.views import add_generic_context
+from hs_core.views.utils import authorize
+from hs_core.models import BaseResource
+from hs_access_control.models import PrivilegeCodes
+
+from .models import CollectionResource
+
+@processor_for(CollectionResource)
+def landing_page(request, page):
+    content_model = page.get_content_model()
+    edit_resource = page_processors.check_resource_mode(request)
+
+    user = request.user
+    if user.is_authenticated():
+        # get a list of resources with effective OWNER privilege
+        owned_resources = user.uaccess.get_resources_with_explicit_access(PrivilegeCodes.OWNER)
```

Comment:
Line 21 - 33: generate a list "user_all_accessible_resource_list" contains resources the current signed-in user has permission (Own, Edit, View), and public or discoverable resources the user discovered from discover page and tagged as "My Favorite"